rclpy/test/test_logging_rosout.py

```diff
@@ -17,20 +17,23 @@
 from rcl_interfaces.msg import Log
 import rclpy
 from rclpy.executors import SingleThreadedExecutor
+from rclpy.logging import LoggingSeverity
 
 TEST_PARAMETERS = [
+    # name, enable_global_rosout_logs, enable_node_rosout, expected_data
     ('enable_global_rosout_enable_node_rosout', True, True, True),
     ('enable_global_rosout_disable_node_rosout', True, False, False),
     ('disable_global_rosout_enable_node_rosout', False, True, False),
     ('disable_global_rosout_disable_node_rosout', False, False, False),
 ]
 
-raw_subscription_msg = None  # None=No result yet
+receive_expected_rosout_msg = False  # True if expected rosout msg is received
 
 
-def raw_subscription_callback(msg):
-    global raw_subscription_msg
-    raw_subscription_msg = msg
+def rosout_subscription_callback(msg):
+    global receive_expected_rosout_msg
+    if (LoggingSeverity(msg.level) == LoggingSeverity.INFO and msg.msg == 'SOMETHING'):
```

Comment:
more precisely checking node name also? I do not think that is necessary.

Reply:
> more precisely checking node name also? I do not think that is necessary.

I think so too.